editor/multi_node_edit.h

```diff
@@ -36,7 +36,7 @@
 class MultiNodeEdit : public RefCounted {
 	GDCLASS(MultiNodeEdit, RefCounted);
 
-	List<NodePath> nodes;
+	Vector<NodePath> nodes;
```

Comment:
Use `LocalVector` for stuff that isn't meant to be passed around in functions.

Reply:
Nevermind, it's used for `SceneTreeDock::get_singleton()->set_selection()`.

Reply:
No you were right this can use LocalVector `SceneTreeDock::get_singleton()->set_selection()` use `Vector<Node>` because `EditorSelection` needs nodes and not nodepath so we need to retrieve nodes anyway ad it was already done there so I just defined the variable a bit earlier to keep it in scope longer. All in all this means this can be replaced with `LocalVector`